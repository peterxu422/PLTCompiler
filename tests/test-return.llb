void main() {
	print(retIntLit());
	print(retIntVar());
	
	print(retDblLit());
	print(retDblVar());
	
	print(retPitchLit());
	print(retPitchVar());
	
	print(retSoundLit());
	print(retSoundVar());
}

int retIntLit() {
	return 1;
}

int retIntVar() {
	int i;
	i = 1;
	return i;
}

double retDblLit() {
	return 1.5;
}

double retDblVar() {
	double d;
	d=1.5;
	return d;
}

pitch retPitchLit() {
	return A5;
}

pitch retPitchVar() {
	pitch p;
	p = A5;
	return p;
}

sound retSoundLit() {
	return A5:1/4:100;
}

sound retSoundVar() {
	sound s;
	s = A5:1/4:100;
	return s;
}
