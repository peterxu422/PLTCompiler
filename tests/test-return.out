1
1
1.5
1.5
A5
A5
A5:1/4:100
A5:1/4:100
